index.d.ts

```diff
@@ -24,6 +24,7 @@ declare namespace nock {
 
   export interface Options {
     allowUnmocked?: boolean;
+    reqheaders:Object
```

Comment:
I see. I used the index signature because it's what I wanted to do.
It is true that primitives string coerce to String and would be seen as an Object.
Sorry I know that Object was so permisive but I didn't know that there was a better way to specify this case.
Thanks